### PR TITLE
Add TC Remove Azure Module and Generate Code

### PR DESCRIPTION
**Pull request by an-phung**

### CODE REVIEW

File: swtbot_test/src/testsuites/ChangeRTOSVersion.java [hunk 1]
```diff
@@ -37,18 +40,12 @@ public static void beforeClass() throws Exception {
...
-		if(list.size()==1) {
-			for(ProjectModel model:list) {
-				projectModelSpecific=model;
-			}
-		}
+		projectModelSpecific = PGUtility.prepareProjectModel(RTOSType.AZURE, RTOSVersion.Azure_6_2_1, RTOSApplication.AZURE_USBX_CDC, Constants.CCRX_TOOLCHAIN, TargetBoard.BOARD_CK_RX65N);
 	}
 	
 	@Test
 	public void tc_01_CreateUsbxProject() throws Exception{
```
Comment (giangchau95) [inline swtbot_test/src/testsuites/ChangeRTOSVersion.java:47]: E: the TC only checks for components after changing the version, it does not show that rtos version is actually changed.
Need to check version too.

File: swtbot_test/src/utilities/Utility.java [hunk 2]
```diff
@@ -142,11 +142,31 @@ public static void addComponentAndGenerate(String componentName) {
 		bot.sleep(15000);
 	}
 	
+	public static void removeComponentAndGenerate(String componentName) {
```
Comment (giangchau95) [inline swtbot_test/src/utilities/Utility.java]: E: separate this utility into two different utils for removing components and generating code.

File: swtbot_test/src/testsuites/RemoveAzureModuleAndGenerateCode.java [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+			assertFalse(true);
+		}
+	}
+	@Test
+	public void tc_04_AddComponentNetxduoAddons() throws Exception{
+		bot.editorByTitle(projectModelSpecific.getProjectName() + ".scfg").setFocus();
+		Utility.addComponentAndGenerate("netx duo addons");
+		Utility.getProjectExplorerView().setFocus();
+		SWTBotTreeItem project = bot.tree().getTreeItem(projectModelSpecific.getProjectName() + " ["+ projectModelSpecific.getActiveBuildConfiguration() +"]");
+		project.getNode(ProjectParameters.FolderAndFile.FOLDER_LIBS).expand();
```
Comment (giangchau95) [inline swtbot_test/src/testsuites/RemoveAzureModuleAndGenerateCode.java]: E: too complicated, no need to migrate 100% from rcptt, check for component added in component tree is enough

**Review by giangchau95 — COMMENTED** [review]
Review-Time: 10m

File: swtbot_test/src/testsuites/RemoveAzureModuleAndGenerateCode.java (hunk 3 above)
Comment (an-phung) [reply swtbot_test/src/testsuites/RemoveAzureModuleAndGenerateCode.java]: Fixed in latest commit

File: swtbot_test/src/utilities/Utility.java (hunk 2 above)
Comment (an-phung) [reply swtbot_test/src/utilities/Utility.java]: Fixed in latest commit

File: swtbot_test/src/testsuites/ChangeRTOSVersion.java (hunk 1 above)
Comment (an-phung) [reply swtbot_test/src/testsuites/ChangeRTOSVersion.java:47]: Fixed in latest commit

**Review by giangchau95 — APPROVED** [review]
Review-Time: 3m

**Pull request merged**